src/getting_started.jl

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
I suggest to move this outside of HerbSearch, but add it as a notebook to the Herb website.